PyDSS/pyPostprocessor/pyPostprocessAbstract.py

```diff
@@ -31,12 +31,15 @@ class AbstractPostprocess(abc.ABC):
     def __init__(self, project, scenario, inputs, dssInstance, dssSolver, dssObjects, dssObjectsByClass, simulationSettings, logger):
         """This is the constructor class.
         """
-        if simulationSettings["Project"]["Simulation Type"] != "Snapshot":
-            raise InvalidParameter("Upgrade post-processors are only supported on Snapshot simulations")
+        #if simulationSettings["Project"]["Simulation Type"] != "Snapshot":
```

Comment:
Should this be deleted?  Or is the rule still valid?

Reply:
Post process can run in any mode. The DERMS controller for examnples runs in QSTS mode not snapshot mode. I have removed this check already.